ast.c

```diff
@@ -334,22 +351,20 @@ static const ast_flag_info flag_info[] = {
 	{ ZEND_AST_CONDITIONAL, 1, conditional_flags },
 };
 
-// NOTE(tandre) in php 8, to get a writeable pointer to a property, OBJ_PROP_NUM(AST_CACHE_SLOT_PROPNAME) can be used.
-
-static inline void ast_update_property(zval *object, zend_string *name, zval *value, void **cache_slot) {
+static inline void ast_update_property(zval *object, zend_string *name, zval *value) {
 #if PHP_VERSION_ID < 80000
-	zval name_zv;
-	ZVAL_STR(&name_zv, name);
-	Z_OBJ_HT_P(object)->write_property(object, &name_zv, value, cache_slot);
+       zval name_zv;
+       ZVAL_STR(&name_zv, name);
+       Z_OBJ_HT_P(object)->write_property(object, &name_zv, value, NULL);
 #else
-	Z_OBJ_HT_P(object)->write_property(Z_OBJ_P(object), name, value, cache_slot);
+       Z_OBJ_HT_P(object)->write_property(Z_OBJ_P(object), name, value, NULL);
```

Comment:
Space indentation snuck in here...

Reply:
Fixed, deleted and manually restored while working on cleaning up cache slots